lib/avtp_pipeline/openavb_common/avb.c

```diff
@@ -69,12 +69,12 @@ int pci_connect(device_t *igb_dev)
 		printf("attaching to %s\n", devpath);
 		err = igb_attach(devpath, igb_dev);
 		if (err) {
-			printf("attach failed! (%s)\n", strerror(err)); /*errno or err ??*/
+			printf("attach failed! (%s)\n", strerror(err));
 			continue;
 		}
 		err = igb_attach_tx(igb_dev);
 		if (err) {
-			printf("attach_tx failed! (%s)\n", strerror(err)); /*errno or err ??*/
+			printf("attach_tx failed! (%s)\n", strerror(err));
```

Comment:
The igb library returns system error codes. I don't see anything wrong with this approach. If we want to create our own set of user codes that's fine, but, in my opinion unnecessary. I think an intermediate approach might be to just wrap strerror in igb_strerror() and in the future it could be changed a little more easily.